CHANGELOG

```diff
@@ -1,5 +1,7 @@
 * 2.12.0 (2019-XX-XX)
 
+ * added the "country_name", "currency_name", "currency_symbol", "language_name", "locale_name",
+   "timezone_name", "country_timezones", "format_currency", "format_number", "format_number_*", "format_date" filters
```

Comment:
missing `format_datetime` and `format_time`

Reply:
fixed